docs/diff.md

```diff
@@ -22,7 +22,8 @@ If `--output` is provided then a report will be written with any differences bet
 
 See <a href="/examples/release-diff.txt" target="_blank">`release-diff.txt`</a> for an example. In the output, 'Ontology 1' corresponds to your `--left` input and 'Ontology 2' corresponds to your `--right` input.
 
-The output is in OWL Manchester syntax, but you can include entity labels with `--labels true`.
```

Comment:
@balhoff Please confirm:

The old documentation was wrong, because the output was always Functional, not Manchester. So the behaviour is not changing, and this PR is backwards compatible.

Reply:
Yes, that's right, the output was always Functional.